flexible/java-11/cloudstorage/pom.xml

```diff
@@ -72,6 +72,52 @@
       <type>jar</type>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <version>5.0.1</version>
+      <scope>test</scope>
+  </dependency>
+    <dependency>
+      <groupId>org.junit.vintage</groupId>
+      <artifactId>junit-vintage-engine</artifactId>
+      <scope>test</scope>
+      <version>4.12.3</version>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.13.2</version>
```

Comment:
I believe there's an effort to remove these package versions, in this github repo. You could search recent PRs for that effort.

Reply:
Here it is: https://github.com/GoogleCloudPlatform/java-docs-samples/pull/7863#pullrequestreview-1383765023